### PR TITLE
[CI] Windows wheels and more improvements 

### PR DESCRIPTION
**Pull request by roigcarlo**
**Description**
The PR updates some of the scripts for the windows wheels in order to reuse core elements.
Also I have reorganzied a bit the json of the applications and moved them along with the applications themselves.

I marking this as a draft as there are some scripts that still need work and some options removed that should be re-enabled. If you have comments, please go ahead with them.

I laso added some scripts so the launch of the containeriez app for the releases is strighforward in windws, has the hyperV backend introduces some non trivial restrictions and limits that need to be disabled in order to utilize the full power of the cluster in which the container is running.

Edit: @HubertBalcerzak If I made a terrible mistake please tell me, I am still "noob" using these techs,

**Changelog**
- Updating windows wheel build process
- Moving app and core json descriptions to application folders (shared for linux and windows)
- Updated the setup script to allow wildcards
- Added container launch scripts for windows (under hyperV) and linux (under WSL2) for windows.

### CODE REVIEW

File: applications/ExternalSolversApplication/ExternalSolversApplication.json
Comment (philbucher) [inline applications/ExternalSolversApplication/ExternalSolversApplication.json]: as this app is deprecated for very long already I would remove it from the wheels

File: applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json
Comment (philbucher) [inline applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json:2]: should this be in the wheels?
I would say no as it is only for the project (isn't it over by now BTW?)

File: kratos/KratosMultiphysics.json
Comment (philbucher) [inline kratos/KratosMultiphysics.json:4]: is that all?
What about e.g. tinyexpr which was added some time ago?

File: applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json
Comment (roigcarlo) [reply applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json:2]: To be honest (as you can image) I generated all this automatically. I know I removed a couple manually but if there is no interest into this to be released as an app we can safely remove it. @riccardotosi ?

File: kratos/KratosMultiphysics.json
Comment (roigcarlo) [reply kratos/KratosMultiphysics.json:4]: As far as I can tell you it does not generate a separated .so / .dll, so no need to add it here.

But since you mention this, I think mmg / hdf5 app should ship with the third party *.so. I will have those fixed thx!

File: applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json
Comment (riccardotosi) [reply applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json:2]: The project finishes on 30/11/2021. About having ExaquteSandboxApplication wheel, it depends on deciding if we want to be able or not run ExaQUte related examples using wheels or not. Who should decide?

File: kratos/KratosMultiphysics.json
Comment (philbucher) [reply kratos/KratosMultiphysics.json:4]: for me it generates `libtinyexpr.a` (on linux)

Ah maybe not on Win?

File: applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json
Comment (philbucher) [reply applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json:2]: ah ok if it is automatic then sure

I am commenting as there is usually an outcry if we remove stuff, even if it was temporary to begin with

I dont mind having it it, your call

File: kratos/KratosMultiphysics.json
Comment (roigcarlo) [reply kratos/KratosMultiphysics.json:4]: a but that is the static lib, it's a byproduct of the compilation process, no need to include it.

File: applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json
Comment (roigcarlo) [reply applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json:2]: Well in this case, we can leave it there. the `json` is not in any specific format or anything, I think @HubertBalcerzak just put the fields he needed so the `setup.py` works.

My idea was to reuse those to add more info about the apps if we ever need to, like if there are research applications, core, etc...

**Comment by roigcarlo** [comment]
Ok, ready for review. This should work for both Windows and Linux without:
FemToDem
PfemFluid
CableNet
HDF5

I need some more underground work for HDF5 and all the others have something wrong with Windows but I will fix it in separate Pr. 

Ping @Rbravo555 and @SADPR as they are also interested in the win part.

File: scripts/wheels/build_release_linux.ps1
Comment (roigcarlo) [inline scripts/wheels/build_release_linux.ps1:1]: Change default branch to master

File: scripts/wheels/build_release_windows.ps1
Comment (roigcarlo) [inline scripts/wheels/build_release_windows.ps1:1]: Change default branch to master

File: scripts/wheels/linux/build.sh
Comment (roigcarlo) [inline scripts/wheels/linux/build.sh:2]: All Pythons should be active in the final release.

File: scripts/wheels/windows/build.ps1
Comment (roigcarlo) [inline scripts/wheels/windows/build.ps1:3]: All Pythons should be active in the final release.

**Comment by maceligueta** [comment]
I see that the description inside the json file of each application is the same, talking about Kratos and not specifically about the app. Is that correct or will we have to change the message in each app?

**Comment by roigcarlo** [comment]
I auto-generated them, so now all they look very much the same. You can change them as you like and add/remove authors, contact info, etc...

If you want you can push directly to this branch :+1:

**Comment by AlejandroCornejo** [comment]
> 
> 
> Ok, ready for review. This should work for both Windows and Linux without:
> FemToDem
> PfemFluid
> CableNet
> HDF5
> 
> I need some more underground work for HDF5 and all the others have something wrong with Windows but I will fix it in separate Pr.
> 
> Ping @Rbravo555 and @SADPR as they are also interested in the win part.

hmm What do you mean? I'll have to adapt something? THX

**Comment by roigcarlo** [comment]
No, no, but github now sends notifications to all team maintainers and since I touched all Kratos files....

**Comment by AlejandroCornejo** [comment]
ok!!!! thank you @roigcarlo

**Comment by roigcarlo** [comment]
Can someone approve this? @KratosMultiphysics/technical-committee

**Review by philbucher — APPROVED** [review]

**Pull request merged**
